src/core/core.controller.js

```diff
@@ -933,7 +937,65 @@ module.exports = function(Chart) {
 			me.lastActive = me.active;
 
 			return changed;
-		}
+		},
+		/**
+		 * format the data list if the data is missing some points in the datasets.
+		 * @private
+		 * @param {array} dataArray array to format
+		 * @return {array} the formated array
+		 */
+		formatDataset: function(dataArray) {
+			var labels = this.chart.data.labels;
+			var tmp = dataArray.slice(0);
+			var labelLen = labels.length;
+			var dataLen = dataArray.length;
+			var result = dataArray;
+			var match = this.dataInLabel(tmp, labels);
+			if (match && dataLen < labelLen) {
+				for (var i = 0; i < labels.length; i++) {
+					var label = labels[i];
+					result[i] = {x: label, y: this.getY(label, tmp)};
+				}
+			}
+			return result;
+		},
+		/**
+		* return the y data of the label, if no this label, return null
```

Comment:
The spacing is a bit off on the doc for this comment and the next one.

They should probably be more like the first method doc you have like:
```
/**
 *
```

This one is missing a space:
```
/**
*
```

Reply:
The missing space is added now in all the places.